XCTestPlans/AWSIntegrationTestsOnCI.xctestplan

```diff
@@ -0,0 +1,95 @@
+{
```

Comment:
This file defines the test targets and the configuration to be used when running integration tests.

Reply:
This file is actually ignored by Git, and you have to "force" Git to commit it with `git add -f`.

It must remain ignored, so that Git credentials aren't accidentally committed.